web/templates/poll_modal_option.hbs

```diff
@@ -0,0 +1,8 @@
+<li class='option-row'>
```

Comment:
I'm surprised the linter didn't flag this, but let's stick with double-quotes around attribute values (and singles around the translation strings) :)

Reply:
Understood the conventions, and fixed this template.

However I also observed few other instances of such violations in other templates which aren't being flagged. Just thought I'd bring that to notice

Reply:
Cool, well, you can join me on Team Consistency to try and fix those things as they pop up :)